docs/resources/Channel.md

```diff
@@ -388,30 +388,28 @@ See [permissions](#DOCS_TOPICS_PERMISSIONS/permissions) for more information abo
 
 Receiving:
 
-| Field     | Type      | Description               |
-| --------- | --------- | ------------------------- |
-| id        | snowflake | role or user id           |
-| type      | string    | either "role" or "member" |
-| allow     | integer   | legacy permission bit set |
-| allow_new | string    | permission bit set        |
-| deny      | integer   | legacy permission bit set |
-| deny_new  | string    | permission bit set        |
+| Field | Type      | Description                   |
+|-------|-----------|-------------------------------|
+| id    | snowflake | role or user id               |
+| type  | int       | either 0 (role) or 1 (member) |
```

Comment:
3 times in this file it says `int` but the rest of the docs say `integer`

Reply:
We are so inconsistent in this. Main docs use integer, sdk uses int :( I'll rectify this in some other commit so it's not literally every file touched.